dockstore-webservice/src/test/java/io/swagger/api/impl/ToolsImplCommonTest.java

```diff
@@ -264,6 +268,31 @@ public void convertDockstoreWorkflowToTool() throws IOException {
         convertDockstoreWorkflowToTool(null, true);
     }
 
+    @Test
```

Comment:
Good idea, should also add a test using the new Galaxy yml descriptor s just in case

Reply:
I'm not sure if that's relevant to the function.  Both would have DescriptorLanguage.FileType.DOCKSTORE_GXFORMAT2 but with different paths.